src/reporters/tiny.js

```diff
@@ -0,0 +1,66 @@
+import path from "path";
+
+export default report;
+
+async function report(runningTests, startTime) {
+  console.log("");
+  const summary = await reportSequentially(runningTests);
```

Comment:
I like that the choice to report sequentially or not got shifted into the reporter 👍 nice extension point.